### PR TITLE
Add proxy_help message

### PR DESCRIPTION
**Pull request by antonsoroko**
Add "Configure proxy in elementum in the same way as here[CR]to avoid issues with downloading torrent files." help message to avoid configuration issues.

continuation of https://github.com/elgatito/script.elementum.burst/pull/407

### CODE REVIEW

File: resources/language/messages.pot [hunk 1]
```diff
@@ -410,3 +410,7 @@ msgstr ""
 msgctxt "#32150"
 msgid "Example: 'HDR, .?265, Hindi'. Regexp are supported.[CR]You can filter languages, video/audio features, etc."
 msgstr ""
+
+msgctxt "#32151"
+msgid "Configure proxy in elementum in the same way as here[CR]to avoid issues with downloading torrent files."
```
Comment (elgatito) [inline resources/language/messages.pot:415]: `elementum` should be `Elementum`

Comment (antonsoroko) [reply resources/language/messages.pot]: @elgatito fixed

**Comment by elgatito** [comment]
There were conflicts, so I added it manually with a new message ID. Thanks

**Comment by antonsoroko** [comment]
thanks! i was away from PC so was not able to resolve conflicts.

**Pull request closed without merging**